src/components/SellerTag.js

```diff
@@ -1,36 +1,95 @@
 import styled from 'styled-components';
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import { SVG } from './svg';
 
-const SellerTag = props => {
+const tagList = [
+  {
+    tagId: 'radio1',
+    tagName: '전체',
+    value: 'All',
+    svgName: SVG.FRUIT,
+  },
+  {
+    tagId: 'radio2',
+    tagName: '냉장',
+    value: 'COLD',
+    svgName: SVG.REFRIGERATOR,
+  },
+  {
+    tagId: 'radio3',
+    tagName: '냉동',
+    value: 'FROZEN',
+    svgName: SVG.FROZEN,
+  },
+  {
+    tagId: 'radio4',
+    tagName: '건조',
+    value: 'DRY',
+    svgName: SVG.DRY,
+  },
+  {
+    tagId: 'radio5',
+    tagName: '국내산',
+    value: 'DOMESTIC',
+    svgName: SVG.MAP,
+  },
+  {
+    tagId: 'radio6',
+    tagName: '수입산',
+    value: 'IMPORTED',
+    svgName: SVG.IMPORT,
+  },
+];
+const SellerTag = ({ sellerList, tagFilter }) => {
+  // useEffect(() => {
+  //   inputTag.current.setAttribute('checked', true);
+  // }, []);
+
+  const [tagInit, setTagInit] = useState('All');
+  const tagSend = e => {
+    tagFilter(e.target.value);
+  };
+  const handleTagInit = () => {
+    setTagInit(null);
+  };
   return (
-    <SellerTagWrap>
-      <SellerTagTitle>여기가 바로 과일 맛집!!!</SellerTagTitle>
+    <SellerTagWrap sellerList={sellerList}>
+      {sellerList !== undefined && (
+        <SellerTagTitle>여기가 바로 과일 맛집!!!</SellerTagTitle>
+      )}
       <SellerTagBox>
-        <SellerTags>{SVG.REFRIGERATOR}냉장</SellerTags>
-
-        <SellerTags>
-          {SVG.FROZEN}
-          냉동
-        </SellerTags>
-        <SellerTags>{SVG.DRY}건조</SellerTags>
-        <SellerTags>
-          {SVG.MAP}
-          국내산
-        </SellerTags>
-        <SellerTags>{SVG.IMPORT}수입산</SellerTags>
+        {tagList.map((item, index) => (
+          <div key={index}>
+            <TagInput
+              type="radio"
+              id={item.tagId}
+              value={item.value}
+              name="radio"
+              onClick={e => {
+                tagSend(e);
+                handleTagInit();
+              }}
+              // ref={item.ref}
+              checked={tagInit === item.value ? true : null}
```

Comment:
- `tagInit === item.value` 라는 expression 자체가 boolean 값이기 때문에 추가적으로 비교할 필요가 없습니다.
```suggestion
              checked={tagInit === item.value}
```